channels/rdpdr/client/rdpdr_main.c

```diff
@@ -430,19 +431,25 @@ static UINT handle_hotplug(rdpdrPlugin* rdpdr)
 		if (device_ext->path == NULL)
 			continue;
 
+		ConvertFromUnicode(CP_UTF8, 0, device_ext->path, 0, &path, 0, NULL, FALSE);
```

Comment:
I guess we should check the return value here. If conversion fails, it may not allocate `path` and further calls to strstr() will segfault.